docker-compose.yaml

```diff
@@ -0,0 +1,46 @@
+version: "3.3"
+services:
+  star-tides:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    environment:
+      FLASK_DEBUG: 1
+      PYTHONUNBUFFERED: 1
+    #env_file:
+    #  - .env
+    restart: always
+    volumes:
+      - ./:/app
+    depends_on:
+      - mongodb_container
+    ports:
+      - 5000:5000
+  test:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    environment:
+      FLASK_DEBUG: 1
+      PYTHONUNBUFFERED: 1
+    restart: always
+    volumes:
+      - ./:/app
+    depends_on:
+      - mongodb_container
+    ports:
+      - 5000:5000
+    command: bash -c "cd /app && pytest star_tides/tests/tests"
+  mongodb_container:
+    image: mongo:latest
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: root
```

Comment:
This is for the test instance only, right?

Reply:
Correct. This is only for local development.